docs/web-sdk/subscribe-to-realtime.md

```diff
@@ -0,0 +1,56 @@
+---
```

Comment:
This file should be deleted. It contains Web SDK details which we are not pushing right now (as mentioned earlier).